.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.217.1/containers/docker-existing-dockerfile
+{
+	"name": "recetox-aplcms-dev",
+
+	// Sets the run context to one level up instead of the .devcontainer folder.
+	"context": "..",
+
+	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+	"dockerFile": "../dockerfile",
```

Comment:
I don't use devcontainers but I am curious why we don't use the standard `Dockerfile` file name?

Reply:
I just didn't realize the standard naming involves capital D. Fixed in 37d0541.

Reply:
Yeah, it is a bit weird, but on some systems this could be an issue. Thanks for changing it.